README.md

```diff
@@ -14,45 +14,45 @@ Completely free, open-source, cross-platform, and blazingly fast asynchronous en
 
 https://noisestudio.net/
 
-> WARNING:<br>
-NoiseEngine is still in VERY EARLY development. Frequent changes, missing docs, and instability may occur.<br>
-You SHOULD avoid using it for any game development before v0.1.0 was released. But [contributions](#how-can-i-contribute) are welcome.
+> **Warning**
+NoiseEngine is still in **very early** development. Frequent changes, missing docs, and instabilities may occur.<br>
+You should avoid using it for **any** game development before v0.1.0 is released. [Contributions](#how-can-i-contribute) are welcome!
 
 ## Why?
-Popular game engines was created many years ago based on synchronous API which does not use all power of modern CPU's. When we creating own next generation procedural game prototype in 2021 with existing engine we started having difficulties to optimize internal things in engine which we used. We started looking for a new engine, but they all had the same problems, so we rethought them and made the difficult decision to create our own engine - NoiseEngine.
+Popular game engines were created many years ago and are based on synchronous APIs which do not take full advantage of modern CPUs. While building the prototype of our own next generation procedural game in 2021 using an existing engine, we had difficulties with optimizing internal things in the engine which we used. We started looking for alternatives, but they all had the same problems, so we made the difficult decision to create our own engine - NoiseEngine.
 
-Our golden goal is create most performant engine, which relatively easy to game developers create games which we currently cannot do. Also in AAA scale.
+Our golden goal is to create the most performant engine, one which enables game developers to create games which are hard to do with current generation engines. Also on AAA scale.
 
-NoiseEngine is not a story of 9th generation of consoles. Is not a story of primitive tool like current engines. Is a story of future with new perspective.
+Our goal is not to create an engine for the current generation of consoles. Our goal is to look even further, and create a bright future for the next generation of developers.
```

Comment:
After rewording this sentence, I think it is so wrong and maybe it should just be removed?